MovieChecker/MovieChecker/AppDelegate.h

```diff
@@ -0,0 +1,17 @@
+//
+//  AppDelegate.h
+//  MovieChecker
+//
+//  Created by Joseph Goldberg on 1/21/15.
+//  Copyright (c) 2015 Vokal. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface AppDelegate : UIResponder <UIApplicationDelegate>
+
+@property (strong, nonatomic) UIWindow *window;
+
+
```

Comment:
You probably should set up Objective-Clean on this project so that it can warn you about things like this extra blank line here.